src/anemoi/utils/commands/remove-metadata.py

```diff
@@ -0,0 +1,47 @@
+# (C) Copyright 2024 Anemoi contributors.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+
+import shutil
+from argparse import ArgumentParser
+from argparse import Namespace
+
+from anemoi.utils.checkpoints import remove_metadata
+
+from . import Command
+
+
+class RemoveMetadata(Command):
+    """Delete the metadata from a checkpoint and create a new checkpoint with the metadata removed."""
+
+    def add_arguments(self, command_parser: ArgumentParser) -> None:
+        """Add arguments to the command parser.
+
+        Parameters
+        ----------
+        command_parser : ArgumentParser
+            The argument parser to which the arguments will be added.
+        """
+        command_parser.add_argument("--input", help="A path to the checkpoint file containing the metadata.")
+
+        command_parser.add_argument("--output", help="Path to checkpoint without metadata")
```

Comment:
What about using source & target instead of input and output, for consistency within the package?